src/main/java/org/squiddev/plethora/api/method/LuaList.java

```diff
@@ -22,8 +22,11 @@
 
 	/**
 	 * Creates a {@link Collector} that produces a {@link LuaList}.
+	 *
+	 * @deprecated Prefer {@link LuaList#of(Collection, Function)}
 	 */
 	@SuppressWarnings({"unchecked", "rawtypes"})
+	@Deprecated
```

Comment:
Can we remove this deprecation? It's still useful in more complex situations:

https://github.com/SquidDev-CC/plethora/blob/f6769030724feddd6fc6f811f2ff68bfbfaa0587/src/main/java/org/squiddev/plethora/integration/forestry/MethodsAnalyzer.java#L42-L47

Still probably worth leaving the "Prefer ..." comment, just not `@deprecated`.

Reply:
That's fine.  I thought that the code in `MethodsAnalyzer` could probably be converted, but I didn't try, since that would be outside the scope of this PR.  The `@Deprecated` annotation also helped me find cases where I could switch to using `LuaList.of(...)`.